### DISCUSSION TITLE
BUTTON page buttons malfunction (?)

### DISCUSSION THREAD

**Discussion by gdercze** [Blackymas/NSPanel_HA_Blueprint#1592] [Q&A]
I just installed HA and NSPanel_HA_Blueprint. I use Sonoff Basic switches (via eWelink extension) and own hw. boards based on ESP01 (via ESPHome extension).
Settings were made on NSPanel page according to the Blueprint's description.
But I have issues with BUTTON page functionality:
Displaying the actual status of an entity - like a Sonoff switch - works fine. But switching a device by pressing the specific icon on NSPanel's touchscreen works just in case of ESP module and does not work on Sonoff devices.

**Comment by edwardtfn** [comment]
- #1598
